app/latigo/log.py

```diff
@@ -56,3 +58,7 @@ def setup_logging(filename, log_level=logging.INFO):
     logger = logging.getLogger(log_filename)
     logger.info(f"Log started for {log_filename}")
     return logger
+
+def add_azure_logging(logger, enable_azure_logging=False, azure_monitor_instrumentation_key=None):
+    if enable_azure_logging:
```

Comment:
Offer to check if key was passed. It might be not obvious error sometimes:
 
```suggestion
    if enable_azure_logging:
        if not azure_monitor_instrumentation_key:
            raise ValueError("'azure_monitor_instrumentation_key' can not be empty if Azure logging is enabled")

```